packages/Testing/PHPUnit/PlatformAgnosticAssertions.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Testing\PHPUnit;
+
+use Nette\Utils\FileSystem;
+use PHPUnit\Framework\Constraint\IsEqual;
```

Comment:
this cause error in prefixed rector

https://github.com/rectorphp/rector/runs/3337580361#step:6:12

Reply:
I created PR https://github.com/rectorphp/rector-src/pull/693 for it.

Reply:
thx @samsonasik 

is this something I could have covered before merging the PR?

Reply:
it is fixed now, you can test scoping by run sh ./full_build.sh locally